### PR TITLE
Extract installCliPlugin function

### PR DESCRIPTION
**Pull request by janisz**
To improve readability I extracted function that installs CLI plugin.

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

File: pkg/cmd/pkg/package_install.go [hunk 1]
```diff
@@ -104,56 +104,59 @@ func pkgInstall(ctx api.Context, packageName string, opts pkgInstallOptions) err
...
+	})
+	if err != nil {
+		return fmt.Errorf("cannot install plugin: %s", err)
+	}
+	plural := ""
+	cmds := plugin.CommandNames()
+	if len(cmds) > 1 {
+		plural = "s"
+	}
+	fmt.Fprintf(ctx.ErrOut(), "New command%s available: dcos %s\n", plural, strings.Join(cmds, ", "))
```
Comment (bamarni) [inline pkg/cmd/pkg/package_install.go]: When there are multiple commands, the message would be `New commands available: dcos foo, bar`. We should use the same message pattern than https://github.com/dcos/dcos-cli/blob/cd01a5d079b4f0eb7024c9085c12965c78fdec01/pkg/cmd/plugin/plugin_add.go#L30 which I think is clearer.

Comment (janisz) [reply pkg/cmd/pkg/package_install.go]: Fixed

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Review by bamarni — APPROVED** [review]

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes @rgo3  Can you review, please?

**Review by drewvanstone — APPROVED** [review]

**Pull request merged**